cmd/terraform.go

```
@@ -70,74 +34,16 @@ func Contains(slice []string, target string) bool {
-func terraformRun(cmd *cobra.Command, actualCmd *cobra.Command, args []string) error {
-	var argsAfterDoubleDash []string
-	var finalArgs = args
-
-	doubleDashIndex := lo.IndexOf(args, "--")
-	if doubleDashIndex > 0 {
-		finalArgs = lo.Slice(args, 0, doubleDashIndex)
-		argsAfterDoubleDash = lo.Slice(args, doubleDashIndex+1, len(args))
+func terraformRun(cmd *cobra.Command, actualCmd *cobra.Command, args []string) {
+	info := getConfigAndStacksInfo("terraform", cmd, args)
+	if info.NeedHelp {
+		actualCmd.Usage()
```

golangci-lint (1.62.2)

Error return value of `actualCmd.Usage` is not checked

(errcheck)